tests/integration/suite/sentry/validator/kubernetes/longname.go

```diff
@@ -82,6 +83,7 @@ func (l *longname) Setup(t *testing.T) []framework.Option {
 		return sentry.New(t,
 			sentry.WithWriteConfig(false),
 			sentry.WithKubeconfig(kubeAPI.KubeconfigPath(t)),
+			sentry.WithMode(string(modes.KubernetesMode)),
```

Comment:
Could you add a self-hosted mode test? And additionally a test for without `KUBERNETES_SERVICE_HOST` to ensure behavior - I only see tests with `KUBERNETES_SERVICE_HOST` set.

Reply:
please check if the integration test I added is enough :)